packages/jss-plugin-global/package.json

```diff
@@ -36,7 +36,8 @@
     "check-snapshot": "node ../../scripts/match-snapshot.js"
   },
   "devDependencies": {
-    "jss-plugin-nested": "10.8.2"
+    "jss-plugin-nested": "10.8.2",
+    "jss-v10_8_0": "npm:jss@10.8.0"
```

Comment:
this looks strange

Reply:
What does strange mean?
- Doesn't work well?
- Not good idea?

Removing tests with older jss is OK with me.